wiki/Music_theory/Time_signature/de.md

```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+tags:
+  - sheet music
+  - Notenblatt
+---
+
+# Taktart
+
+*Für mehr Informationen über Taktarten in der Musiktheorie, siehe: [Taktangabe](https://de.wikipedia.org/wiki/Taktangabe)*
+
+Eine **Taktart** ist ein musikalischer Begriff, der die Anzahl an [Beats](/wiki/Music_theory/Beat) in einem einzigen [Takt](/wiki/Music_theory/Measure) eines Songs (die obere Zahl), und welche Note einen einzelnen Beat markiert (die untere Zahl), definiert. Bei einem 4/4-Takt (auch bekannt als "common time") passen 4 1/4 Noten (*Viertelnoten*) in einen Takt, wohingegen ein 3/4-Takt (auch bekannt als "Waltzer" oder "Dreivierteltakt") bedeutet, dass 3 1/4 Noten in einen Takt passen.
+
+In osu! bietet der [Beatmap-Editor](/wiki/Client/Beatmap_editor) mehrere Taktarten für [Mapping](/wiki/Beatmapping) an, die im [Timing-Menü](/wiki/Client/Beatmap_editor/Timing) ausgewählt werden können. Dazu gehören 3/4, 4/4, 5/4, 6/4 und 7/4.
```

Comment:
`für Mapping` doesn't feel correct. `fürs Mapping`? Or maybe remove the `Mapping` reference altogether to avoid this: `mehrere Taktarten an, die ...`

Reply:
```suggestion
In osu! bietet der [Beatmap-Editor](/wiki/Client/Beatmap_editor) mehrere Taktarten beim [Mapping](/wiki/Beatmapping) an, die im [Timing-Menü](/wiki/Client/Beatmap_editor/Timing) ausgewählt werden können. Dazu gehören 3/4, 4/4, 5/4, 6/4 und 7/4.
```

Does this sound better?

Reply:
sure